packages/launchpad/src/types.ts

```diff
@@ -1,7 +1,7 @@
 export interface Task {
 	id: string;
-	completed: boolean;
-	disabled: boolean;
+	completed?: boolean;
+	disabled?: boolean;
```

Comment:
Is this type change strictly necessary? I am not sure why we need this change.

Reply:
I'm not sure why that's changed TBH! I changed it back and TypeScript seems satisfied, maybe it was a Type error in a previous iteration.